### DISCUSSION TITLE
Using Oh my posh without Windows EXE, only psm1

### DISCUSSION THREAD

**Discussion by mkanet** [JanDeDobbeleer/oh-my-posh#4424] [General]
@JanDeDobbeleer or anyone, My work just adopted a new policy that doesn't allow 3rd party EXEs on our Windows PCs anymore.  

Is there any way I can use my current Oh-my-posh custom theme file `mkanet.omp.json` with:  https://github.com/JanDeDobbeleer/oh-my-posh2 ?

If not, is there a special version of oh-my-posh that doesn't require the EXE (just like the one from the link above) and also allows me to use my custom theme?

**Comment by JanDeDobbeleer** [comment]
@mkanet no, the exe is always necessary. That said, oh-my-posh is available from the Store, is that an option?

**Reply by mkanet** [reply]
Thanks for your reply.  That's too bad, the EXE wasn't required in previous versions of Oh My Posh. Unfortunately the Microsoft Store isn't available to us at work.

I would just use the older version or Oh My Posh that didn't require the EXE, but it only supports builtin themes.  I couldnt figure out how to adapt my custom theme code meant for the EXE version of Oh My Posb to a native custom theme for the old Oh My Posh.

**Reply by JanDeDobbeleer** [reply]
> the EXE wasn't required in previous versions of Oh My Posh

Yes, but that's 4 years ago by now. The current version has nothing in common with the old one except for the name. I can't be held responsible for paranoid IT departments.
